package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "internship",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
```

Comment:
Not saying that it's bad practice to use caret range in dependencies but let's do not use it for this project just to avoid any possible package changes

Reply:
Ok, i got it